tests/testthat/helper-tests.R

```diff
@@ -38,6 +71,147 @@ test_compare_csv_db <- function(con, csv, db) {
               label = "DB Compare: Data matches")
 }
 
+compare_csv <- function(res, tables) {
+  expect_true(all(unlist(lapply(seq_along(tables), function(ti) {
+    table <- tables[ti]
+    test_compare_csv_db(
+      res$con, res$e[[paste0(table, "_csv")]], table)}))))
+}
+
+mess_with <- function(path, csv, col, row, text) {
+  data <- read.csv(file.path(path, "meta", csv),
+                   stringsAsFactors = FALSE)
+  if (row == 0) {
+    names(data)[names(data) == col] <- text
+  } else {
+    data[[col]][row] <- text
+  }
+  write.csv(data, file.path(path, "meta", csv), row.names = FALSE)
+}
+
+db_file <- function(db, f) {
+  if (!db) f
```

Comment:
As a style point I usually do this either with braces or on exactly one line as

```
if (!db) f else paste0("db_", f)
```

Reply:
Yep, that's nicer.